libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -176,6 +176,9 @@
     border:none;
 }
 
+/** Hide the modal body by default **/
+.jevbootstrap > div.action_dialogJQ  {display:none;}
```

Comment:
It varies on sites templates, but by default we do not set a display until we call it which it's a display block. So this just makes sure it's hidden until it is called.